### ISSUE TITLE
Request: 2024 cores

### ISSUE THREAD

**Issue by mario-64** [MrLightgun/MiSTerSindenDriver#5]
Would it be possible to get Sinden-compatible cores using the most recent as baseline? The included ones appear to be over a year old.
PSX core, for example, was just updated in April. 
Thanks again though for even what we have now. Got mine working tonight. So very cool.  :-)

**Comment by trislett** [comment]
Agreed. Or maybe forking the original core repositories? I think they are GPL 3.0, so any modified cores need to be open source anyways. It would also help with pull requests.

**Comment by MrLightgun** [comment]
Hi @mario-64 and @trislett , yes definitely, I'm waiting for a chance for the dev who did the core changes to share that code so I can publish it and hopefully then get a version built with the latest updates.  Hopefully that might also show how the SegaCD and Master system cores could be updated too with a border.  

I would prefer that the border could just be a menu option similar to crosshair on/off so hopefully that's an option at some point too.

**Comment by MrLightgun** [comment]
Sources for the cores now added against the latest cores.  I'm currently rebuilding and testing.

**Comment by wally86** [comment]
Can the MegaCD Sinden core be updated to newest release? I'm getting some audio hiccups during snatcher. (CD Audio)
